modules/flowable-engine/src/main/java/org/flowable/engine/impl/bpmn/behavior/UserTaskActivityBehavior.java

```diff
@@ -18,6 +18,8 @@
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import static java.util.stream.Collectors.toList;
```

Comment:
Thanks for the PR.

This is a *very* small issue given the work done here but the project standards call for all `static` imports to be first in the file.  Obviously this doesn't change the execution of the code but is just the chosen style.

Thanks again.

Reply:
@dbmalkovsky thanks for the review I have done the change.